src/sentry/auth/helper.py

```diff
@@ -158,6 +159,8 @@ def handle_existing_identity(
         sample_rate=1.0,
     )
 
+    if request.user and not is_active_superuser(request):
```

Comment:
i think checking `request.user` is redundant, we do it when we set activeorg in endpoint code but i think its always exists here.

Reply:
its possible for request.user to be None technically speaking

Reply:
(right, mean not `None`)